docs/docs/developer/style.md

```diff
@@ -0,0 +1,27 @@
+# Style
+
+## Python
+
+To ensure a consistent Python code style, we use [Ruff](https://docs.astral.sh/ruff/) for both linting and formatting.
```

Comment:
Maybe pin the version here or link to the poetry file.

Reply:
I added a link to the poetry file a line or two below. Is it enough?

Manually keeping the version up to date wouldn't go too well imo and they don't have a version dropdown or tags like we do.